iib/workers/tasks/utils.py

```diff
@@ -24,6 +24,7 @@
 from iib.exceptions import IIBError, AddressAlreadyInUse
 from iib.workers.config import get_worker_config
 from iib.workers.api_utils import set_request_state
+from iib.workers.tasks.fbc_utils import is_image_fbc, _get_index_fbc_dir
```

Comment:
the function `_get_index_fbc_dir` should be without `_` at the beginning if we import it here

Reply:
can we resolve this one? It is no longer private function